embedx/config.schema.json

```diff
@@ -347,8 +347,14 @@
                 "file:///etc/configs/keto_namespaces.ts",
                 "ws://my.websocket.server/keto_namespaces.ts"
               ]
+            },
+            "experimental_strict_mode": {
```

Comment:
@zepatrik I would rather just name this `strict_mode`. In my mind, we should make `strict_mode` the default in the (somewhat distant) future, and maybe soon have it as the default for new projects.

This means that soon enough, we would have `experimental_strict_mode` and `strict_mode` flags in the config and we would need to decide which to take. WDYT?

Reply:
SGTM

Reply:
Alright, makes sense, I already have some ideas to make it even more strict ;)